app/src/main/res/values-ar/strings.xml

```diff
@@ -48,225 +47,122 @@
-    <string name="app_components_badge_type_label">النوع</string>
-    <string name="app_components_badge_standardType_label">قياسي</string>
-    <string name="app_components_badge_countType_label">عدد</string>
-    <string name="app_components_badge_iconType_label">أيقونة</string>
-    <string name="app_components_badge_count_label">عدد</string>
+    <string name="app_components_badge_description_text">Badge هي عنصر صغير في واجهة المستخدم يُستخدم لتسليط الضوء على الحالة أو الإشعارات أو التصنيف داخل الواجهة. غالبًا ما يتم عرضها كعلامة أو مؤشر بلون خلفية مميز ونص.</string>
 
     <!-- Components: button -->
-    <string name="app_components_button_label">زر</string>
-    <string name="app_components_button_description_text">الأزرار تسمح للمستخدمين باتخاذ قرارات أو أداء إجراء. لها عدة أنماط لتلبية احتياجات مختلفة.</string>
+    <string name="app_components_button_description_text">تسمح Buttons للمستخدمين باتخاذ قرارات أو تنفيذ إجراءات. وتتوفر بأنماط متعددة لتلبية احتياجات مختلفة.</string>
 
     <!-- Components: checkbox -->
-    <string name="app_components_checkbox_label">مربع اختيار</string>
-    <string name="app_components_checkbox_description_text"> تتيح خانات الاختيار للمستخدمين تحديد خيار واحد أو عدة خيارات من قائمة، أو تفعيل/تعطيل إعدادات، أو تأكيد إجراء معين.</string>
-    <string name="app_components_checkbox_checkbox_label">خانة اختيار</string>
-    <string name="app_components_checkbox_checkboxItem_label">عنصر خانة اختيار</string>
+    <string name="app_components_checkbox_description_text">تتيح Checkboxes للمستخدمين تحديد خيار واحد أو خيارات متعددة من القائمة، أو تشغيل الإعدادات أو إيقاف تشغيلها، أو تأكيد إجراء ما.</string>
     <string name="app_components_checkbox_checkbox_a11y">وصف محتوى مربع الاختيار %s</string>
-    <string name="app_components_checkbox_indeterminateCheckbox_label">خانة اختيار غير محدد</string>
     <string name="app_components_checkbox_indeterminateCheckbox_a11y">وصف محتوى مربع الاختيار غير المحدد %s</string>
-    <string name="app_components_checkbox_indeterminateCheckboxItem_label">خانة اختيار اختيار غير محدد</string>
 
     <!-- Components: chip -->
-    <string name="app_components_chip_label">عنصر</string>
-    <string name="app_components_chip_description_text">تساعد الـعناصر  المستخدمين على إدخال المعلومات، أو القيام باختيارات، أو تصفية المحتوى، أو تنفيذ إجراءات.</string>
-    <string name="app_components_chip_filterChip_label">نصر تصفية</string>
+    <string name="app_components_chip_description_text">تساعد Chips   المستخدمين على إدخال المعلومات، القيام باختيارات، تصفية المحتوى، أو تنفيذ إجراءات.</string>
     <string name="app_components_chip_filterChip_chipContent_label">تصفية</string>
-    <string name="app_components_chip_suggestionChip_label">عنصر اقتراح</string>
     <string name="app_components_chip_suggestionChip_chipContent_label">اقتراح</string>
```

Comment:
These are the only labels that are translated in demo screens. I suppose we should not translate them?

<img width="275" height="542" alt="Capture d’écran 2025-12-16 à 15 11 12" src="https://github.com/user-attachments/assets/46474b9e-f516-4f98-9a67-c532da16b112" />

Reply:
Actually, these are the only components where we don't display "label" as content. As iOS and Flutter display "label", I suggest to use `app_components_common_label_label` here.